atd-vze/src/Components/Notes/Notes.js

```diff
@@ -10,17 +10,18 @@ import {
   Button,
 } from "reactstrap";
 import moment from "moment";
-import { notesDataMap } from "./notesDataMap";
-import {
+import { notesDataMap } from "./notesDataMap.js";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare a notes component
+const Notes = ({
+  crashLocationId,
+  tableName,
```

Comment:
The Notes component accepts tableName and various query/mutation functions as parameters because the Location Notes and Crash Notes pull from different tables in the DB.

Reply:
I'm curious if anyone would have approached this differently or can think of a more obvious way to make the component reusable between crashes/locations.